### PR TITLE
Add Prettier

### LINKED ISSUE #121: Add prettier
https://github.com/prettier/prettier
https://github.com/prettier/eslint-plugin-prettier
https://github.com/prettier/eslint-config-prettier

### PR DESCRIPTION
**Pull request by vanessa**
## Description

Add Prettier (with a config file `.prettierrc.json`) to the project so developers won't have to worry about code format.

Minor: Format `CONTRIBUTING.md` (into a checklist-ish)

Closes #121.

## Motivation and Context

By using Prettier, people spend less time making sure their code is well formatted and more time developing actual features. The config file is also customizable, so people will also have flexibility to change if needed.

### Prettier
1. Create a new copy of the boilerplate doing [these steps](https://github.com/vintasoftware/django-react-boilerplate/blob/master/CONTRIBUTING.md#how-to-test-django-admin-startproject)
1. Run `npm install` and create a new JavaScript file with this code (beware!):
    ```jsx
    import React from 'react'
    import PropTypes from 'prop-types'


    const MyComponent = ({
    children
    }) => (<div>Hello world, this is my unformatted code <p>{children}</p> <div>One element</div> <div>Other element</div></div>)


    MyComponent.propTypes = {children: PropTypes.node.isRequired}

    export default MyComponent
    ```
1. Run the command in your preferred text editor to format the code. The output will be:
    ```jsx
    import React from 'react';
    import PropTypes from 'prop-types';

    const MyComponent = ({ children }) => (
      <div>
        Hello world, this is my unformatted code <p>{children}</p> <div>One element</div>{' '}
        <div>Other element</div>
      </div>
    );

    MyComponent.propTypes = { children: PropTypes.node.isRequired };

    export default MyComponent;
    ```
1. Run the frontend and check if there are any errors

### Changes to CONTRIBUTING.md
- Check if they make sense with the previous instructions

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires documentation updates.
- [x] I have updated the documentation
- [x] My change requires dependencies updates.
- [x] I have updated the dependencies accordingly.

### CODE REVIEW

File: .prettierrc.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "printWidth": 100,
+    "useTabs": false,
+    "tabWidth": 2,
+    "semi": false,
```
Comment (victorfsf) [inline .prettierrc.json]: Not sure if we should disable semicolons and trailing commas here, but i believe this is something that'd require input from others.

Comment (vanessa) [reply .prettierrc.json]: @chocoelho Do you have any thoughts on it?

**Comment by vanessa** [comment]
@victorfsf Done that. And added semicolons and trailing commas to the configuration file. Now they're showing on eslint.

**Comment by joaopslins** [comment]
Wouldn't this be deprecated in favor of our new linting tools, like the plugin? @vanessa @victorfsf

**Comment by victorfsf** [comment]
Yes, but the edits in CONTRIBUTING.md are worth saving.

**Comment by victorfsf** [comment]
This PR is outdated. @vanessa plz extract the minor changes to the `CHANGELOG.md` to another PR.

**Pull request closed without merging**